pkg/rds.go

```diff
@@ -483,7 +498,15 @@ func (e *RDSExporter) addAllLogMetrics(ctx context.Context, sessionIndex int, in
 	wg.Wait()
 }
 
-func (e *RDSExporter) addAllInstanceMetrics(sessionIndex int, instances []*rds.DBInstance) {
+func (e *RDSExporter) addAllInstanceMetrics(sessionIndex int, instances []*rds.DBInstance, eolInfos []EOLInfo) {
+	// eolMap keys EOLInfo by engine and version
```

Comment:
Inline comment should be used to describe the reason instead of syntax (why instead of what/how), if can't tell the intention of the code, extract it to a function, no need to comment on things already expressed by language syntax.